src/types/service_config.go

```diff
@@ -2,6 +2,10 @@ package types
 
 import "github.com/Originate/exosphere/src/util"
 
+type DevelopmentConfig struct {
+	Scripts map[string]string `yaml:",omitempty"`
```

Comment:
Thoughts on Scripts being a struct with Run and Test properties instead of a map of string to string?

Reply:
I figured the scripts would be user specifiable. Any script you want to run in the dev docker image should go here much how `npm`/`yarn` do it. I imagine there being cases where we want to run other scripts in the dev docker image and we shouldn't restrict the user to just those two use cases

Reply:
We don't support something like that at the moment though and I'm also skeptical what that would be used for. I'd like to see a use case before we consider it

Reply:
It would be useful for things like reinstalling dependencies on nodejs / go and it would also be useful for people who want to hack on services written in languages they do not have compilers/runtimes installed

Reply:
But I'm also ok making this a struct until we need it to be a map

Reply:
Also it would be good to have the option of splitting up lint, unit-tests and cucumber tests into separate commands